Add Translation Program mission and vision section (#4046)
ethereum/ethereum-org-website: src/content/contributing/translation-program/index.md

- [Content versions](/contributing/translation-program/content-versions/) _– Which pages are included in each content version of ethereum.org_

## Mission and vision {#mission-and-vision}

The Ethereum community strives to be truly global and all-inclusive. Given that over 3/4 of the world’s population do not speak English, we want to provide them with content in their native language, in order for ethereum.org to be the portal to Ethereum for the global community.

> The aim of the ethereum.org Translation Program is to make Ethereum accessible to everyone, regardless of their nationality and language, by translating the website and other Ethereum content into as many languages as possible.

Reviewer: ```suggestion
The ethereum.org Translation Program aims to make Ethereum accessible to everyone by translating ethereum.org and other Ethereum content into as many languages as possible.
```

Reviewer: >  regardless of their nationality and language

This is already implied

**Our mission**

> - Provide translated versions of the website to empower visitors from all over the world to learn about Ethereum in their native language.

Reviewer: ```suggestion
- Provide translated versions of the website to empower visitors worldwide to learn about Ethereum in their native language
```

> - Facilitate the onboarding of those who stand to benefit the most from Ethereum

Reviewer: > Facilitate the onboarding of those who stand to benefit the most from Ethereum

This seems a bit hand-wavy to me. Who benefits the most?

> - Allow for easier and more inclusive sharing of Ethereum information and knowledge.

Reviewer: ```suggestion
- Allow for more accessible and more inclusive sharing of Ethereum information and knowledge
```

> - Open up the Translation Program to voluntary contributions to empower community members from all over the world to have a chance to contribute to Ethereum and make their mark on the ecosystem.

Reviewer: ```suggestion
- Empower community members to contribute translations to Ethereum and make their mark on the ecosystem
```

> Open up the Translation Program to voluntary contribution

I think this is pretty well implied already

> - Identify and connect with supporters and contributors looking to get involved with Ethereum and help provide guidance.

Reviewer: ```suggestion
- Identify, connect with, and provide guidance to passionate contributors looking to get involved in the ecosystem
```

**Our vision**

> - Translate essential content for members of the Ethereum community from as many countries and parts of the world as possible.

Reviewer: > essential content

How do we define essential content?

Reviewer: On second read I'm okay with this being vague 👍

> - Support knowledge sharing across languages with the aim of creating a better informed and educated Ethereum community.

Reviewer: ```suggestion
- Support knowledge sharing across languages to create a better informed and educated Ethereum community
```

> - Increase Ethereum's inclusivity and accessibility by providing translated content and removing the language barriers that prevent non-English speakers from joining the ecosystem.

Reviewer: ```suggestion
- Increase Ethereum's inclusivity and accessibility by removing the language barriers preventing non-English speakers from joining the ecosystem
```

**Latest updates**

> To keep in touch with the latest Translation Program progress, you can follow the [Ethereum Foundation blog](https://blog.ethereum.org/).
> 
> Our most recent posts are available below:
> 
> - [August 2019 Translation Program launch](https://blog.ethereum.org/2019/08/20/translating-ethereum-for-our-global-community/)
> - [July 2020 milestones update](https://blog.ethereum.org/2020/07/29/ethdotorg-translation-milestone/)
> - [December 2020 milestones update](https://blog.ethereum.org/2020/12/21/translation-program-milestones-updates-20/)

Reviewer: ```suggestion
To keep up-to-date with the latest Translation Program progress, you can follow the [Ethereum Foundation blog](https://blog.ethereum.org/):

- [December 2020 milestones update](https://blog.ethereum.org/2020/12/21/translation-program-milestones-updates-20/)
- [July 2020 milestones update](https://blog.ethereum.org/2020/07/29/ethdotorg-translation-milestone/)
- [August 2019 Translation Program launch](https://blog.ethereum.org/2019/08/20/translating-ethereum-for-our-global-community/)
```

IMO the updates should be in reverse chronological order (we already do this elsewhere)

<img width="1239" alt="Screenshot 2021-09-28 at 11 21 03" src="https://user-images.githubusercontent.com/62268199/135069513-b1332074-986f-4ab0-a93a-be6d3dc2e326.png">

> Our most recent posts are available below:

We can probably drop this, it's pretty implied / already has the law of proximity working for it.

- [Content versions](/contributing/translation-program/content-versions/) _– Which pages are included in each content version of ethereum.org_

> ## Translation Program mission and vision {#mission-and-vision}

Reviewer 2: ```suggestion
## Mission and vision {#mission-and-vision}
```

Reviewer 2: Should this be the top section on the page (i.e. in front of "How to translate" & "Resources")?

> The Ethereum community is truly global and all-inclusive. However, as over 3/4 of the world’s population do not speak English, we want to provide them with content in their native language, in order to be the portal to Ethereum for the global community.

Reviewer 2: ```suggestion
The Ethereum community strives to be truly global and all-inclusive. Given that over 3/4 of the world’s population do not speak English, we want to provide them with content in their native language, in order for ethereum.org to be the portal to Ethereum for the global community.
```

The aim of the ethereum.org Translation Program is to make Ethereum accessible to everyone, regardless of their nationality and language.

> To achieve this, our community of volunteer translators is working on translating the website and other Ethereum content into as many languages as possible.

Reviewer 2: I think this sentence could be brought into the paragraph above.

**Our mission**

> - By providing translated versions of the website, visitors from all over the world can learn about Ethereum in their native language.

Reviewer 2: ```suggestion
- Provide translated versions of the website to enable visitors from all over the world to learn about Ethereum in their native language.
```

>     - This helps to bridge language gaps and provides visitors with more inclusive and more generally understandable content.

Reviewer 2: ```suggestion
    - Bridge language gaps and provide visitors with more inclusive and more generally understandable content.
```

>     - In addition, this facilitates the onboarding of more members to the global Ethereum community.

Reviewer 2: ```suggestion
    - Facilitate the onboarding of more members to the global Ethereum community.
```

> - Translated content about Ethereum allows for easier and more inclusive sharing of information and knowledge.

Reviewer 2: ```suggestion
- Allow for easier and more inclusive sharing of Ethereum information and knowledge.
```

> - By opening up the Translation Program to voluntary contributions, community members from all over the world have a chance to contribute to Ethereum and make their mark on the ecosystem.

Reviewer 2: ```suggestion
- Open up the Translation Program to voluntary contributions to empower community members from all over the world to have a chance to contribute to Ethereum and make their mark on the ecosystem.
```

> - By opening up the Translation Program to voluntary contributions, community members from all over the world have a chance to contribute to Ethereum and make their mark on the ecosystem.

Reviewer 3: ```suggestion
- Work with voluntary contributors and community members from all over the world to provide a chance for anyone to contribute to Ethereum and make their mark on the ecosystem.
```

> - The Translation Program also allows us to identify and connect with numerous supporters and contributors, looking to get involved with Ethereum.

Reviewer 2: ```suggestion
- Identify, connect with, and engage numerous supporters and contributors looking to get involved with Ethereum.
```

> - The Translation Program also allows us to identify and connect with numerous supporters and contributors, looking to get involved with Ethereum.

Reviewer 3: ```suggestion
- Identify and connect with supporters and contributors looking to get involved with Ethereum and help provide guidance.
```

**Our vision**

> - To translate essential content for members of the Ethereum community from as many countries and parts of the world as possible.

Reviewer 2: ```suggestion
- Translate essential content for members of the Ethereum community from as many countries and parts of the world as possible.
```

> - To support the sharing of knowledge across languages with the aim of creating a better informed and educated Ethereum community.

Reviewer 2: ```suggestion
- Support knowledge sharing across languages with the aim of creating a better informed and educated Ethereum community.
```

> - To increase Ethereum's inclusivity and accessibility by providing translated content and removing the language barriers that prevent non-English speakers from joining the ecosystem.

Reviewer 2: ```suggestion
- Increase Ethereum's inclusivity and accessibility by providing translated content and removing the language barriers that prevent non-English speakers from joining the ecosystem.
```

> **Ethereum Foundation blog**

Reviewer 2: ```suggestion
**Latest updates**
```

> To keep in touch with the latest updates about the Translation Program, you can follow the [Ethereum Foundation blog](https://blog.ethereum.org/).

Reviewer 2: ```suggestion
To keep in touch with the latest Translation Program progress, you can follow the [Ethereum Foundation blog](https://blog.ethereum.org/).
```

> Our older posts are available below:

Reviewer 2: ```suggestion
Our most recent posts are available below:
```

> - [Translation Program announcement](https://blog.ethereum.org/2019/08/20/translating-ethereum-for-our-global-community/)

Reviewer 2: ```suggestion
- [August 2019 Translation Program launch](https://blog.ethereum.org/2019/08/20/translating-ethereum-for-our-global-community/)
```

> - [July 2020 post](https://blog.ethereum.org/2020/07/29/ethdotorg-translation-milestone/)

Reviewer 2: ```suggestion
- [July 2020 milestones update](https://blog.ethereum.org/2020/07/29/ethdotorg-translation-milestone/)
```

> - [December 2020 post](https://blog.ethereum.org/2020/12/21/translation-program-milestones-updates-20/)

Reviewer 2: ```suggestion
- [December 2020 milestones update](https://blog.ethereum.org/2020/12/21/translation-program-milestones-updates-20/)
```
